hs_core/models.py

```diff
@@ -4228,7 +4227,7 @@ def _create_person_element(self, etree, parent_element, person):
             hsterms_name = etree.SubElement(dc_person_rdf_Description,
                                             '{%s}name' % self.NAMESPACES['hsterms'])
 
-            hsterms_name.text = name_without_commas(person.name)
+            hsterms_name.text = person.name
 
```

Comment:
here and elsewhere, is trim() a consideration?

Reply:
trim() is js, I see python strip() in files below

Reply:
Yeah, good catch, I should add a strip() call before doing the conditional check ```if person.name:``` to capture the potential case with only spaces in person.name.

Reply:
strip() returns a copy of the string, so the strip isn't reflected here.